extensions/hazelcast-3-connector/hazelcast-3-connector-common/src/main/java/com/hazelcast/connector/Hz3Enrichment.java

```diff
@@ -60,7 +60,7 @@
  * config.addCustomClasspaths(name, jars);
  * } </pre>
  * The jars must exist in the directory specified by the
- * {@link com.hazelcast.jet.core.JetProperties#PROCESSOR_CUSTOM_LIB_DIR}
+ * {@link com.hazelcast.spi.properties.ClusterProperty#PROCESSOR_CUSTOM_LIB_DIR}
```

Comment:
Do this and add the import, IDE doesn't do it by default, but it's valid:
```suggestion
 * {@link ClusterProperty#PROCESSOR_CUSTOM_LIB_DIR}
```